### PR TITLE
Align Vega wallet footer items

### LINKED ISSUE #880: Align version with change key and disconnect in vega wallet
Per title

### PR DESCRIPTION
**Pull request by pgriscti**
closes #880

Screenshot 2021-11-17 at 17 32 16

Screenshot 2021-11-17 at 17 32 00

### CODE REVIEW

File: src/components/vega-wallet/vega-wallet.tsx [hunk 1]
```diff
@@ -114,7 +114,7 @@ export const VegaWallet = () => {
           )}
         </WalletCardHeader>
         <WalletCardContent>{child}</WalletCardContent>
-        <WalletCardContent>{version}</WalletCardContent>
+        {!version ? <WalletCardContent>{version}</WalletCardContent> : null}
```
Comment (mattrussell36) [inline src/components/vega-wallet/vega-wallet.tsx:117]: This seems weird. If we have no version we display the version?

File: src/components/vega-wallet/vega-wallet.tsx [hunk 2]
```diff
@@ -389,6 +389,7 @@ const VegaWalletConnected = ({
 
   const disconnect = (
```
Comment (mattrussell36) [inline src/components/vega-wallet/vega-wallet.tsx:390]: Can we rename this to footer or something?
```suggestion
  const footer = (
```

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: src/components/vega-wallet/vega-wallet.tsx (hunk 1 above)
Comment (pgriscti) [reply src/components/vega-wallet/vega-wallet.tsx]: fixed

File: src/components/vega-wallet/vega-wallet.tsx (hunk 2 above)
Comment (pgriscti) [reply src/components/vega-wallet/vega-wallet.tsx]: done

**Review by dexturr — APPROVED** [review]

**Pull request merged**